blockauth/utils/token.py

```
@@ -314,67 +314,34 @@ def generate_auth_token_with_custom_claims(
-    except ImportError as e:
-        # Fall back to original implementation if enhanced system is not available
-        logger.error(f"❌ ImportError in enhanced JWT system: {e}")
-        logger.error(f"❌ ImportError type: {type(e)}")
-        import traceback
-
-        traceback.print_exc()
-        logger.warning("⚠️ Falling back to original implementation due to ImportError")
+    except ImportError:
+        logger.warning("Enhanced JWT system not available, using fallback")
         return generate_auth_token(token_class, user_id, user_data)
     except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)